extensions/openai/completions.py

```diff
@@ -104,7 +104,7 @@ def process_parameters(body, is_legacy=False):
         # XXX convert tokens from tiktoken based on requested model
         # Ex.: 'logit_bias': {'1129': 100, '11442': 100, '16243': 100}
         try:
-            encoder = tiktoken.encoding_for_model(generate_params['model'])
+            encoder = tiktoken.encoding_for_model(generate_params['model'] if generate_params['model'] else "llama")
```

Comment:
Good question. This passes the model name and not the type to tiktoken. So in the case that the "model" param was loaded it would be some string. Let's say it's AuroraNights or Euryale.. that's all tiktoken gets. Have you peeked into the place it fails and what tiktoken matches "startswith" on?

Reply:
Whoever wrote this must have assumed it was going to be used with openAI and it's pulling double duty. 

so perhaps it's better to just write `if not if generate_params['model'] pass` or something like that.

I will have to test and try it.